### DISCUSSION TITLE
How can I see version number of my Flowise installed?

### DISCUSSION THREAD

**Discussion by artemdzyuba** [FlowiseAI/Flowise#797] [Q&A]
When I go to About Flowise, I see the latest release version, but have no clue what version I have.
How can I find it?
![image](https://github.com/FlowiseAI/Flowise/assets/20184727/ba5ea1fa-e467-4b5d-ac03-34498c692657)

**Comment by HenryHengZJ** [comment]
Will implement that I'm next release!
